elyra/pipeline/component_parser_kfp.py

```diff
@@ -13,15 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import os
-from typing import List
-
 import yaml
+from typing import List
```

Comment:
Per the linter's suggestion

```suggestion

from typing import List
```